common/Stress/Azure.Test.Stress/StressProgram.cs

```diff
@@ -44,6 +44,26 @@ public static async Task Main(Assembly assembly, string[] args)
 
         private static async Task Run(Type testType, StressOptions options)
         {
+            // Test types must contain a public constructor SampleTest(SampleOptions, SampleMetrics)
+            Type metricsType = null;
+            foreach (var ctor in testType.GetConstructors())
+            {
+                var parameters = ctor.GetParameters();
+                if (parameters.Length == 2 &&
```

Comment:
Why prohibit having other parameters with defaults? It may be nice to have the freedom to require a public constructor with the first two arguments of the expected type and allow for additional customizations for local runs. Granted, if there are no defaults, the test will fail when the constructor is invoked, but, to me, that seems like a reasonable sharp edge to allow for an internal tool.